charger/ocpp.go

```diff
@@ -152,6 +141,21 @@ func NewOCPP(id string, connector int, idtag string,
 		if err := ocpp.Instance().Register(id, cp); err != nil {
 			return nil, err
 		}
+
+		// fix timing issue in EVBox when switching OCPP protocol version
+		time.Sleep(time.Second)
+
+		log.DEBUG.Printf("waiting for chargepoint: %v", connectTimeout)
+
+		select {
+		case <-time.After(connectTimeout):
```

Comment:
Do we still need separate connectTimeout?

Reply:
It is better to keep it.
But the name should be changed, because connectTimeout is actually the term in the OCPP environment for the permissible waiting time between the establishment of the cable connection and the successful authentication.
But this is about establishing the OCPP connection via websocket.